web/src/app/modules/actions/modules/browser-version-checker/components/browser-version-checker.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+  moduleId: module.id.toString(),
+  selector: 'browser-version-checker',
+  templateUrl: 'browser-version-checker.component.html',
+  styleUrls: ['browser-version-checker.component.css']
+})
+
+
+
+export class BrowserVersionChecker {
+
+  private unsupportedBrowser = [
+    { browser: 'IE', minVersion: 11 },
+    { browser: 'MSIE', minVersion: 10 },
+    { browser: 'Chrome', minVersion: 42 },
+    { browser: 'Firefox', minVersion: 44 },
+    { browser: 'Edge', minVersion: 14 },
+    { browser: 'Safari', minVersion: 9 },
+    { browser: 'Opera', minVersion: 29 }];
```

Comment:
I was thinking about writing it into the config-file. But actually it should not be changeable. What do you think? Where's the better place?

Reply:
I think the config file should be the best place. Because we might want to update this to newer browser versions in the future.